scripts/deploy/deployScoutGameErc20.ts

```diff
@@ -0,0 +1,88 @@
+import { execSync } from 'node:child_process';
+import fs from 'node:fs';
+import path from 'node:path';
+
+import dotenv from 'dotenv';
+import { task } from 'hardhat/config';
+import { createPublicClient, createWalletClient, encodeDeployData, http } from 'viem';
+import { privateKeyToAccount } from 'viem/accounts';
+
+import { getConnectorFromHardhatRuntimeEnvironment, getConnectorKey } from '../../lib/connectors';
+
+dotenv.config();
+
+const PRIVATE_KEY = (
+  process.env.PRIVATE_KEY?.startsWith('0x') ? process.env.PRIVATE_KEY : `0x${process.env.PRIVATE_KEY}`
+) as `0x${string}`;
+
+task('deployScoutGameERC20', 'Deploys or updates the BuilderNFTSeasonOne contracts').setAction(
```

Comment:
TODO: We should add an initialise() method or something similar in the ERC20 so that we don't have ability to mint new tokens from thin air